src/CC.hs

```diff
@@ -12,7 +12,6 @@ module CC
 
 import Data.Aeson (ToJSON(..), (.=), encode, object)
 import Data.Char (toUpper)
-import Data.Monoid ((<>))
```

Comment:
Why touch this module at all?

Reply:
Ah! Suppress a warning that exhibits with 8.8.1!